Libs/MRML/DisplayableManager/vtkMRMLCrosshairDisplayableManager.cxx

```diff
@@ -285,10 +281,6 @@ void vtkMRMLCrosshairDisplayableManager::vtkInternal::BuildCrosshair()
   // Remove the old actor is any
   if (this->Actor.GetPointer())
   {
-    if (this->LightBoxRenderer)
```

Comment:
Shouldn't this be 

```
  if (this->External->GetRenderer() && this->External->GetRenderer()->HasViewProp(actor))
  {
       this->External->GetRenderer()->RemoveActor(this->Actor);
  }
```

Alternatively, should the actor ever be deleted outside the dtor ?